docker-compose-elk.yml

```
@@ -0,0 +1,29 @@
+version: "3"
+
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.6.2
```

```suggestion
    image: elasticsearch:7.17.10
```
기존에 발생했던 성능저하는 하드웨어 단계에서 일어난 이미지 플랫폼 이슈인 것 같습니다.
arm image를 사용하면 성능이 개선됩니다.

---

오 좋은데요 ㅎㅎ 푸시해주시죵